### PR TITLE
Changed game title and re-added workflow

### PR DESCRIPTION
**Pull request by JasonLovesDoggo**

### CODE REVIEW

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
```
Comment (n0remac) [inline .github/workflows/codeql-analysis.yml:12]: This is a vulnerability scan. As a security engineer I approve, but do we really want to wait for this to run each pr?

Alternatively we could run it on a daily schedule?

Comment (JasonLovesDoggo) [reply .github/workflows/codeql-analysis.yml:12]: ile set it to a cron sced

File: assets/configs/config.ini [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [WINDOW]
-FPS = 60
-TITLE = title
+FPS = 10
```
Comment (n0remac) [inline assets/configs/config.ini:2]: Ultimately, I think we will go for smoother game play. 60 is better for now I think.

File: .github/workflows/codeql-analysis.yml (hunk 1 above)
Comment (n0remac) [reply .github/workflows/codeql-analysis.yml:12]: this should work
```
on:
  push:
    branches: [ main ]
  pull_request:
    # The branches below must be a subset of the branches above
    branches: [ main ]
  schedule:
    - cron: '0 0 * * *'
```

**Comment by JasonLovesDoggo** [comment]
Imma make a new pr with updated code

**Pull request closed without merging**